subsys/net/lib/http_server/src/headers/index_html_gz.h

```diff
@@ -0,0 +1,16 @@
+unsigned char src_index_html_gz[] = {
```

Comment:
This file should be generated at build time. Please remove it with `git rm`, and then `git commit --fixup <commit>`, followed by a `git rebase -i --autosquash <prev_commit>`